packages/inter-protocol/test/psm/test-psm.js

```diff
@@ -426,3 +426,45 @@ test('metrics', async t => {
     },
   });
 });
+
+test('wrong give giveStableInvitaion', async t => {
+  const {
+    zoe,
+    feeMintAccess,
+    initialPoserInvitation,
+    terms,
+    installs: { psmInstall },
+    anchor,
+  } = t.context;
+
+  const mockChainStorage = makeMockChainStorageRoot();
+
+  /** @type {Awaited<ReturnType<import('../../src/psm/psm.js').start>>} */
+  const { publicFacet } = await E(zoe).startInstance(
+    psmInstall,
+    harden({ AUSD: anchor.issuer }),
+    { ...terms },
+    harden({
+      feeMintAccess,
+      initialPoserInvitation,
+      storageNode: mockChainStorage.makeChildNode('thisPsm'),
+      marshaller: makeBoard().getReadonlyMarshaller(),
+    }),
+  );
+
+  const giveAnchor = AmountMath.make(anchor.brand, 200n * 1_000_000n);
```

Comment:
Please use the driver. Almost everything above this line can be `const { publicFacet } = makeDriver()`

Reply:
This intentionally uses the wrong currency in its offer to verify the `fit` call. I didn't see how to get access to the context (for mints, etc.) via the driver. The driver also wouldn't allow me to give `Anchor` while calling `swapStableForAnchor`.

Reply:
The driver provides a `publicFacet` escape hatch. Here's what I meant by "almost everything above this line":
```js
  const { zoe, anchor } = t.context;

  const { publicFacet } = await makePsmDriver(t);
```
Then the rest can stay the same. Though I'd recommend using NonNullish instead of the expect-error because it will alert if another error pops up:
```js
        harden({ In: NonNullish(anchor.mint).mintPayment(giveAnchor) }),
```

Reply:
Altogether,
```js
test('wrong give giveStableInvitaion', async t => {
  const { zoe, anchor } = t.context;

  const { publicFacet } = await makePsmDriver(t);

  const giveAnchor = AmountMath.make(anchor.brand, 200n * 1_000_000n);

  await t.throwsAsync(
    () =>
      E(zoe).offer(
        E(publicFacet).makeGiveStableInvitation(),
        harden({ give: { In: giveAnchor } }),
        harden({ In: NonNullish(anchor.mint).mintPayment(giveAnchor) }),
      ),
    {
      message:
        'proposal: required-parts: give: In: brand: "[Alleged: aUSD brand]" - Must be: "[Alleged: IST brand]"',
    },
  );
});
```

Reply:
done.  Thanks.